zstacklib/zstacklib/utils/ovn.py

```
@@ -299,3 +306,214 @@ def addUplink(self, portPciMap, bondMode, lacpmode, ip, netmask, brName="br-phy"
+        except shell.ShellError:
+            return True, None
+
+    @bash.in_bash
+    def setOvsOtherConfig(self, key, value):
+        queueCmd = CtlBin + " --no-wait set Open_vSwitch . other_config:{}={}".format(key, value)
+        bash.bash_r(queueCmd)
+
+    @bash.in_bash
+    def getNicRxQueueNumConfig(self, nicName):
+        try:
+            ret = bash.bash_r(CtlBin + "get Interface {} options:n_rxq".format(nicName)).strip("\n").strip('"')
```

Pylint (3.3.7)

[error] Instance of 'int' has no 'strip' member

(E1101)